python/lsst/ap/pipe/make_apdb.py

```diff
@@ -108,8 +116,43 @@ def makeApdb(args=None):
     parser = ConfigOnlyParser()
     parsedCmd = parser.parse_args(args=args)
 
-    apdb = parsedCmd.config.diaPipe.apdb.apply(
-        afw_schemas=dict(DiaObject=make_dia_object_schema(),
-                         DiaSource=make_dia_source_schema()))
+    apdb = Apdb(config=parsedCmd.config,
+                afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                 DiaSource=make_dia_source_schema()))
     apdb.makeSchema()
     return apdb
+
+
+# --------------------------------------------------------------------
+# argparse.Actions for use with ConfigOverrides
+# ConfigOverrides is normally used with Click, but I don't have time
+# to learn it.
```

Comment:
This comment is perhaps a bit too honest... I think it's OK to say ingest uses Click for this, but you chose argparse. The context here is that gen3 no longer has a config override mechanism but we need this functionality for make_apdb, right? You could explicitly say that here, but I suppose having it in the commit message is sufficient.